README.md

```diff
@@ -57,6 +57,7 @@ EXAMPLES:
     refitter ./openapi.json --no-deprecated-operations
     refitter ./openapi.json --operation-name-template '{operationName}Async'
     refitter ./openapi.json --optional-nullable-parameters
+    refitter ./openapi.json --use-polymorphic-serialization
```

Comment:
This line would be the output of defining a [Spectre CLI example](https://spectreconsole.net/cli/commands) in [Program.cs](https://github.com/christianhelle/refitter/blob/67faccd61d19938c55578ad06984d7c4b2e84be2/src/Refitter/Program.cs#L150)

Let's get this in as it is and I can the Spectre CLI example